Dockerfile

```diff
@@ -27,37 +27,35 @@ ENV UMASK=0022
 ENV ALLOW_DIRECT_VNC=false
 
 # Prepare installation
-RUN apt-get -q update
-RUN LC_ALL=C DEBIAN_FRONTEND=noninteractive ${PKG} lsb-release
-
-# Install all the stuff
-RUN LC_ALL=C DEBIAN_FRONTEND=noninteractive ${PKG} \
-    gosu \
-    locales \
-    openbox \
-    supervisor \
-    tigervnc-common \
-    $(lsb_release -sc | grep -q bookworm && echo tigervnc-tools) \
-    tigervnc-standalone-server \
-    tint2 \
-    python3-pip \
-    python3-venv \
-    nginx-light
-
-# Cleanup
-RUN apt-get clean && \
+RUN apt-get -q update > /dev/null && \
+    apt-get -q -y upgrade > dev/null && \
+    LC_ALL=C DEBIAN_FRONTEND=noninteractive ${PKG} apt-utils lsb-release && \
+    LC_ALL=C DEBIAN_FRONTEND=noninteractive ${PKG} \
+        gosu \
+        locales \
+        nginx-light \
+        openbox \
+        python3-pip \
+        python3-venv \
+        supervisor \
+        tigervnc-common \
+        tigervnc-standalone-server \
+        $(lsb_release -sc | grep -q bookworm && echo tigervnc-tools) \
+        tint2 && \
+    apt-get clean && \
     apt-get autoremove && \
     rm -rf /var/lib/apt/lists/* && \
     rm -rf /var/cache/fontconfig/*
 
 # novnc installation and patching
+# then install websockify
+# then cleanup novnc artifacts
 COPY --from=novnc /app/novnc/ /opt/novnc/
-RUN sed -i "s/UI.initSetting('resize', 'off')/UI.initSetting('resize', 'remote')/g" /opt/novnc/app/ui.js
-
-
-# install websockify
-RUN python3 -m venv /opt/websockify
-RUN /opt/websockify/bin/pip3 install websockify
+RUN sed -i "s/UI.initSetting('resize', 'off')/UI.initSetting('resize', 'remote')/g" /opt/novnc/app/ui.js && \
+    python3 -m venv /opt/websockify && \
+    /opt/websockify/bin/pip3 install websockify && \
+    cd /opt/novnc && \
+    rm -rf .git* .eslint* AUTHORS docs README.md tests
```

Comment:
Instead of removing not needed files, I'd recommend only copying what's required. We can definitely make use of [newer options](https://docs.docker.com/reference/dockerfile/#copy---parents) if required!